packages/fxa-settings/src/components/PageDeleteAccount/index.tsx

```diff
@@ -0,0 +1,221 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { FormEvent, useCallback, useEffect, useState } from 'react';
```

Comment:
Use like `useEffect` is unused, as well as `cloneDeep` below:

![Screen Shot 2021-01-13 at 10 19 30 AM](https://user-images.githubusercontent.com/6392049/104471645-1a984780-5589-11eb-944f-478b218f7a4a.png)